packages/core/core-flows/src/order/workflows/order-edit/confirm-order-edit-request.ts

```diff
@@ -133,8 +133,14 @@ export const confirmOrderEditRequestWorkflow = createWorkflow(
       throw_if_key_not_found: true,
     }).config({ name: "order-items-query" })
 
-    const lineItemIds = transform({ orderItems }, (data) =>
-      data.orderItems.items.map(({ id }) => id)
+    const lineItemIds = transform(
+      { orderItems, previousOrderItems: order.items },
+
+      (data) => {
+        const previousItemIds = data.previousOrderItems!.map(({ id }) => id) // items that have been removed with the change
```

Comment:
suggestion: just for certainty instead of doing `previousOrderItems!` would we do `previousOrderItems?` just to not end up in a place where it is not present?